doc/source/developing/releasing.rst

```diff
@@ -201,12 +209,14 @@ issue the following command:
    twine upload dist/*
 
 Please ensure that both the source distribution and binary wheels are present in
-the ``dist`` folder before doing this.
+the ``dist`` folder before doing this. Directions on generating binary wheels 
+are described in the section immediately preceding this one. 
 
 You will be prompted for your PyPI credentials and then the package should
 upload. Note that for this to complete successfully, you will need an account on
-PyPI and that account will need to be registered as an "owner" of the yt
-package. Right now there are five owners: Matt Turk, Britton Smith, Nathan
+PyPI and that account will need to be registered as an "owner" or "maintainer" 
+of the yt package. 
+Right now there are five owners: Matt Turk, Britton Smith, Nathan
```

Comment:
you can add yourself too now

Reply:
I got a notice from PyPI that I was a maintainer (not an owner) so I'm not sure I belong in this list.

Reply:
Maintainers can upload releases too and not everyone on this list is actually an owner. I'd just say "right now the following people have access to upload packages:"

Reply:
Sounds good! I've fixed that now.